### ISSUE TITLE
normalizeDate sets all dates 1 month earlier

### ISSUE THREAD

**Issue by roddik** [f1nnix/node-wxr#1]
.getMonth() in JS returns 0 for January, 1 for February, so it's possible for normalizeDate to write date strings like 2015-00-01, which fails to import to Wordpress

**Comment by f1nnix** [comment]
@roddik 10x, I'll fix and push update to npm ASAP.

**Comment by roddik** [comment]
Thanks, for now I'm using this workaround:

```
        string = string.replace /(<wp:post_date>\d{4}-)(\d{2})/g, (full, start, month) ->
            val = parseInt(month, 10) + 1
            val = '0' + val if val < 10
            start + val
```

**Comment by f1nnix** [comment]
@roddik Should be fixed in 0.1.2. Unfortunately, for now I have no time to write even a few tests, so fixed on-the-fly. Hope it works.

Thank you for contributing.

**Comment by roddik** [comment]
Thanks, seems to be working

On Mon, Dec 28, 2015 at 10:39 PM, Ilya Rusanen notifications@github.com
wrote:

> @roddik https://github.com/roddik Should be fixed in 0.1.2.
> Unfortunately, for now I have no time to write even a few tests, so fixed
> on-the-fly. Hope it works.
> 
> Thank you for contributing.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/f1nnix/node-wxr/issues/1#issuecomment-167649425.

## 

Cheers,
Rodion
